test/COURSE/General/filter.js

```diff
@@ -0,0 +1,61 @@
+import { browser } from "@wdio/globals";
+import AuthAppActions from "../../../lib/util/AuthApp/actions.js";
+import CourseManagementActions from "../../../lib/util/CourseManagement/actions.js";
+
+const AuthApp = new AuthAppActions();
+const CourseManagement = new CourseManagementActions();
+var loginFlag = false;
```

Comment:
I will try to run the tests in an incognito browser and hopefully this won't be needed